### PR TITLE
[ 1.51.0 ] Release Notes

### PR DESCRIPTION
**Pull request by DanStough**

### CODE REVIEW

File: content/release-notes/1.51.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+{{<features>}}
+{{</features>}}
+
+{{<changes>}}
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that caused KOTS to fail to detect private images if their domains are blocked by a firewall in online installs.
+* Fixed a bug that caused the [set config](/kots-cli/set/config/) command to fail if appliction config had a required password value that was already set.
```
Comment (samantha-replicated) [inline content/release-notes/1.51.0.md:16]: application *typo*

File: content/release-notes/1.51.0.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Fixed a bug that caused [LocalImageName](/reference/template-functions/config-context/#localimagename) function to not require private images when `proxyPublicImages` is set.
+* Fixed a bug that could cause the secret used to pull private images to be removed during snapshot restore when multiple applications are installed.
+* Fixed a bug that could cause [Helm render](/vendor/helm/helm-processing/) chart using API not supported by the target kubernetes cluster.
+* Fixed a bug that caused newer style charts with private images failed to kustomize.
```
Comment (samantha-replicated) [inline content/release-notes/1.51.0.md:20]: kustomize >customize *typo*

File: content/release-notes/1.51.0.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<features>}}
+{{</features>}}
+
+{{<changes>}}
+{{</changes>}}
+
+{{<fixes>}}
```
Comment (samantha-replicated) [inline content/release-notes/1.51.0.md:14]: For now, we can follow the existing style convention. But it's redundant under the Bug Fixes category to begin every sentence with "Fixed a bug....". From a UX perspective, that is well defined by the section title. Unless there are objections, I would suggest going straight into meat of each bullet point moving forward. So for this particular RN, if I were writing it, would go....

{{<fixes>}}

- KOTS was failing to detect private images if the domains were blocked by a firewall during online installs.
- The set config command failed when the application config had a required password value that was already set.
- The LocalImageName function did not require private images when `proxyPublicImages` is set.
- The secret used to pull private images was removed during snapshot restore when multiple applications are installed.
- When using an API, the Helm render chart was not supported by the target Kubernetes cluster.
- Newer style charts with private images were failing to kustomize. *See line item question below*
- Newer style charts with private images were coexisting with legacy style charts causing failed deployments. 

{{</fixes>}}

File: content/release-notes/1.51.0.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+* Fixed a bug that caused KOTS to fail to detect private images if their domains are blocked by a firewall in online installs.
+* Fixed a bug that caused the [set config](/kots-cli/set/config/) command to fail if appliction config had a required password value that was already set.
+* Fixed a bug that caused [LocalImageName](/reference/template-functions/config-context/#localimagename) function to not require private images when `proxyPublicImages` is set.
+* Fixed a bug that could cause the secret used to pull private images to be removed during snapshot restore when multiple applications are installed.
+* Fixed a bug that could cause [Helm render](/vendor/helm/helm-processing/) chart using API not supported by the target kubernetes cluster.
+* Fixed a bug that caused newer style charts with private images failed to kustomize.
+* Fixed a bug that caused coexistence of legacy and newer style charts with private images failed to deploy.
```
Comment (samantha-replicated) [inline content/release-notes/1.51.0.md:21]: This one was super confusing to read. I re-worded above with how I understood it. Please clarify if i misunderstood :)

**Review by samantha-replicated — COMMENTED** [review]
Finished my review and left some comments. Happy to look over this again in more depth next week if we're adding more content and you want some fresh eyes/edits :) Just let me know!

**Comment by DanStough** [comment]
Closing this one in favor of #564 .

**Pull request closed without merging**